src/js/utils/merge-options.js

```diff
@@ -3,46 +3,44 @@
  */
 import merge from 'lodash-compat/object/merge';
 
-function isPlain(obj) {
+const isPlain = function(obj) {
```

Comment:
We had been using `var f = function() {}` bindings most everywhere else. I was just reverting this to what I thought was the current recommendation.